src/assets/wise5/authoringTool/project-authoring/project-authoring.component.ts

```diff
@@ -170,4 +170,8 @@ export class ProjectAuthoringComponent {
       this.getSelectedNodeIds().every((nodeId) => this.projectService.isApplicationNode(nodeId))
     );
   }
+
+  protected setNodeTypeSelected(nodeType: NodeTypeSelected): void {
+    this.nodeTypeSelected = nodeType;
+  }
```

Comment:
Do we need this function, or can we just set the variable directly in this class?
```suggestion
```